src/OpenCensus.Exporter.Stdout/OpenCensus.Exporter.Stdout.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'OpenCensus.sln'))\build\Common.prod.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
```

Comment:
If there is nothing blocking this, can you bring it down to `net45` instead of `net46`. I think it will be very useful exporter and we may want to use it quite widely